TeamPage.html

```diff
@@ -419,6 +419,56 @@ <h2>Meet Our Team</h2><br>
             </div>
           </div>
         </div>
+        <div class="col-sm-3 col-xs" style="min-width: 210px;">
+          <div class="id_presentation">
+            <img class="img_staff"
+              src="https://avatars.githubusercontent.com/u/153774845?v=4"
+              style="width: 200px; height: 200px; object-fit: cover; object-position: top;" alt="Zach's Photo" />
+            <div class="text_pres">Zach Dorff<br>
+              <div class="subtext_pres">Student Programmer</div>
+            </div>
+            <div class="container" style="text-align: center;">
+              <button type="button" style="margin-top: 10px;" class="btn btn-danger btn-sm" data-toggle="modal"
+                data-target="#myModalTschida" onclick="openBSModal('myModalTschida', this)">Zach's Profile</button>
+              <div class="modal fade" tabindex="-1" id="myModalTschida" role="dialog" datim-modal>
+                <div class="modal-dialog">
+                  <div class="modal-content">
+                    <div class="modal-header">
+                      <div class="modal-title">
+                        <p class="text-justify"><b>Zach Dorff</b> - Student Programmer<br>
+                          Work Location: Remote<br>
+                          Email: Zachary.Dorff@unlv.edu</p>

```

Comment:
Minor thing here, but could you lowercase your email to match the style of all the other profiles?